source/messaging/managing-channels.rst

```diff
@@ -143,7 +143,10 @@ To unmute the channel, select the channel name again to access the drop-down men
 Switching channels
 ------------------
 
-Use the **Find channel** option in the channel sidebar, or press CMD/CTRL+K, to visit other channels, or to review `member availability <https://docs.mattermost.com/help/getting-started/setting-your-status-availability.html#setting-your-availability>`__ at a glance. 
+Use the **Find channel** option in the channel sidebar, or press CMD/CTRL+K, to visit other channels, or to review `member availability <https://docs.mattermost.com/help/getting-started/setting-your-status-availability.html#setting-your-availability>`__ at a glance.
+
+    .. image:: ../../images/switch-channels.png
```

Comment:
Not sure if this image was intentionally removed, but it doesn't work in preview.

PR for reference: https://github.com/mattermost/docs/pull/4698/files

Reply:
@amyblais Image links need to be updated to ``../images/filename.png`` rather than ``../../images``

Reply:
I've made these updates.